components/EventDetails.js

```diff
@@ -0,0 +1,67 @@
+import React, { PropTypes } from 'react';
+import { DateBlock } from '../components';
+import { Link } from '../routes';
+
+const renderAddress = ({ address_lines: addressLines, locality, region, postal_code: postalCode }) => {
+  // Will have to see how this data structure holds up over different events
+  return (
+    <div>
+      { addressLines[0] && <span>{addressLines[0]}</span> }
+      <span>{locality} {region}, {postalCode}</span>
+    </div>
+  );
+};
+
+const renderTimeRange = (startDate, endDate) => {
+  // TDB how to properly format this - would be nice to avoid bringing in moment.js
```

Comment:
👍  i like [date-fns](https://date-fns.org/) if we do need something heavy-weight, since it doesn't mutate dates